### ISSUE TITLE
Form example warning

### ISSUE THREAD

**Issue by Matt-Butler** [cerner/terra-core#413]
# Issue Description
When submitting a form the following error is shown in the console.

```
Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `ControlledInput`. See https://fb.me/react-warning-keys for more information.
```

![warning](https://cloud.githubusercontent.com/assets/8146159/26601795/e3538a74-4546-11e7-8bca-83dd1af7e5bf.gif)

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior
No warnings on submission.

## Current Behavior
Warning on submission

## Steps to Reproduce

1. Navigate to the form examples
2. Click submit on the last example
3. Warning appears in the console

## Environment

* Component Version: Forms

**Comment by Matt-Butler** [comment]
https://github.com/cerner/terra-core/blob/master/packages/terra-site/src/examples/form/examples/ControlledInput.jsx#L64

We should also update 

```
value={this.state.formDatafirst} 
```
tp
```
value={this.state.formData.first} 
```

**Comment by Matt-Butler** [comment]
In the docs, `helpText` has also been changed to `help`.

**Comment by JakeLaCombe** [comment]
I saw some of theses come up. They should be fixed in this pull request. https://github.com/cerner/terra-core/pull/410

**Comment by JakeLaCombe** [comment]
Looks like this is fixed now that #410 is merged.
